ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,69 @@
+<!--
+************ STOP!! ************
+If this issue identifies a security vulnerability, DO NOT submit it! Instead, contact
+the Apache Software Foundation Security Team at security@trafficcontrol.apache.org and follow the
+guidelines at https://www.apache.org/security/ regarding vulnerability disclosure.
+-->
+
+<!--
+- For *SUPPORT QUESTIONS*, use the
+[Traffic Control slack channels](https://traffic-control-cdn.slack.com) or [Traffic Control mailing lists](http://trafficcontrol.apache.org/mailing_lists/).
+- Before submitting, please **SEARCH GITHUB** for a similar issue or PR. -->
+
+**I'm submitting a ...**
+<!-- (check one with "x") -->
+<!--- security vulnerability (STOP!! - see above)-->
+- [ ] bug report
+- [ ] regression from TC 3.x
```

Comment:
I'd suggest something more like "Something used to work in an older version of Traffic Control, but doesn't anymore (please specify versions)". Otherwise, how do you indicate a regression from 2.x? Or from 4.x once that's been released?